spring-boot-project/spring-boot-actuator-autoconfigure/src/main/java/org/springframework/boot/actuate/autoconfigure/tracing/zipkin/ZipkinConfigurations.java

```diff
@@ -59,15 +65,20 @@ static class SenderConfiguration {
 	static class UrlConnectionSenderConfiguration {
 
 		@Bean
-		@ConditionalOnMissingBean(Sender.class)
-		URLConnectionSender urlConnectionSender(ZipkinProperties properties,
-				ObjectProvider<ZipkinConnectionDetails> connectionDetailsProvider) {
+		@ConditionalOnMissingBean(BytesMessageSender.class)
+		URLConnectionSender urlConnectionSender(ZipkinProperties properties, Encoding encoding,
+				ObjectProvider<ZipkinConnectionDetails> connectionDetailsProvider,
+				ObjectProvider<HttpEndpointSupplier.Factory> endpointSupplierFactoryProvider) {
```

Comment:
this bean doesn't seem to be found in my integration tests, I'll have to dig deeper into what I did wrong here..

Reply:
nevermind it works.. I'll post back!